From: Quanah Gibson-Mount <quanah@openldap.org>
Date: Mon, 3 Jun 2024 15:08:22 +0000
Subject: Remove extraneous quote

Origin: upstream, https://git.openldap.org/openldap/openldap/-/commit/7bc97addc83f109f1e5e203e28362767279d29af
Bug: https://bugs.openldap.org/show_bug.cgi?id=10221
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/openldap/+bug/2072976
Applied-Upstream: yes
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index d231e0e..e00d380 100644
--- a/configure.ac
+++ b/configure.ac
@@ -746,7 +746,7 @@ if test $ol_enable_perl != no ; then
 			PERL_CPPFLAGS="$PERL_CPPFLAGS $opt"
 		done
 		PERL_LDFLAGS=""
-		for opt in `$PERLBIN" -MExtUtils::Embed -e ldopts`; do
+		for opt in `$PERLBIN -MExtUtils::Embed -e ldopts`; do
 			case "$opt" in
 			"-lc" )
 				continue;;
